docs/book/v3/cookbook/debug-toolbars.md

```diff
@@ -69,28 +69,6 @@ return array_merge(ConfigProvider::getConfig(), [
 ]);
 ```
 
-In addition, ensure the [PSR-15 HTTP message factory interfaces](https://www.php-fig.org/psr/psr-15/)
```

Comment:
I think we should still have some notes here, in case someone wants use another PSR-15 implementation.
Now, in case diactoros is used, we just need to register the module (config provider) which is automated as long as component-installer is installed.

Reply:
BTW it should be PSR-17 there - HTTP Factories, not PSR-15 - HTTP Server Request Handlers.

Reply:
I agree with @michalbundyra — on `composer update`, the user is not prompted to install a config provider, so unless a user starts with the skeleton _today_ forwards, they'll still need to add either the config provider (if they update to Diactoros 2.3), or manually register them (based on the package they use for PSR-17).

Reply:
I've fixed the typo fix psr-17 to psr-15 and note for start diactoros 2.3.0, can use its ConfiProvider class